src/custom/utils/price.ts

```diff
@@ -243,6 +242,11 @@ function _checkFeeErrorForData(error: GpQuoteError) {
   }
 }
 
+function formatAtoms(amount: string, decimals: number): string {
```

Comment:
TODO: move to state/orders/priceUtils

I know this is legacy and you are just moving it from cow-js ;)